app/models/jws.py

```diff
@@ -26,8 +26,20 @@ def check_at_least_one_field_is_populated(cls, values):
         did, verification_method = values.get("did"), values.get("verification_method")
         if not did and not verification_method:
             raise CloudApiValueError(
-                "At least one of `did` or `verification_method` must be populated."
+                "One of `did` or `verification_method` must be populated."
             )
+        if did and verification_method:
+            raise CloudApiValueError(
+                "Only one of `did` or `verification_method` can be populated."
+            )
+        return values
+
+    @model_validator(mode="before")
+    @classmethod
+    def check_payload_is_populated(cls, values):
+        payload = values.get("payload")
+        if not payload:
+            raise CloudApiValueError("`payload` must be populated.")
```

Comment:
Let's keep code coverage up and add some basic unit tests to cover this new code. Similar to how it's done in `app/tests/models/test_jsonld.py`